docs/adr/0018-integration-tests-third-party-apis.md

```diff
@@ -0,0 +1,49 @@
+# Mock third-party APIs in CI/CD integration tests
+
+**User Story:** [EASi 688](https://jiraent.cms.gov/browse/EASI-688)
+
+When running integration tests in our CI environment, how will EASi test the
+calls to third-party APIs? Will those API calls be mocked or will they hit the
+live APIs? The main driver behind this ADR is CEDAR integration tests, and a
+desire to sufficiently test EASI-CEDAR integration while providing fast feedback
+in CI. Okta tests are out of scope.
+
+## Considered Alternatives
+
+* Do not mock the API calls
+* Mock the API calls
+
+## Decision Outcome
+
+* Chosen Alternative: Mock the API calls, as this will speed up the tests and
+  reduce operational and networking overhead from the CI environment.
+Additionally, the health check tests that run on server startup already provide a
```

Comment:
I would still advocate for adding a single test into CI to test something similar. The startup tests don't prevent code from being merged if it breaks the API client setup.

Reply:
The startup tests run in the DEV environment during the `deploy_dev` job, which is required before merging. I thought that met the requirement for a single test in CI to test the CEDAR API client.

Reply:
I added some context and elaborated on the startup tests:  [7d84f84](https://github.com/CMSgov/easi-app/pull/300/commits/7d84f840c7e593701c79888170c4c46cb09a3cfb)